flw/src/test/java/org/motechproject/nms/flw/osgi/WhiteListServiceBundleIT.java

```diff
@@ -131,4 +131,38 @@ public void testStateWithWhitelistInvalidNumber() throws Exception {
         boolean result = whitelistService.numberWhitelistedForState(whitelistState, NOT_WHITELIST_CONTACT_NUMBER);
         assertFalse(result);
     }
+
+    // Test state with whitelist, number not in list
```

Comment:
If it's easier, you can just update the comment using the github online editor and commit it.